tools/webidl_binder.py

```diff
@@ -614,12 +614,10 @@ def make_call_args(i):
         # this function comes from an ancestor class; for operators, we must cast it
         cast_self = 'dynamic_cast<' + type_to_c(func_scope) + '>(' + cast_self + ')'
       maybe_deref = deref_if_nonpointer(raw[0])
-      if '=' in operator:
-        call = '(*%s %s %s%s)' % (cast_self, operator, maybe_deref, args[0])
-      elif operator == '[]':
+      if operator == '[]':
         call = '((*%s)[%s%s])' % (cast_self, maybe_deref, args[0])
       else:
-        raise Exception('unfamiliar operator ' + operator)
+        call = '(*%s %s %s%s)' % (cast_self, operator, maybe_deref, args[0])
```

Comment:
Can we error somehow if this is not one of the allowed operators? For example if it is a unary operator. Before this PR we'd error on unsupported things, and it would be good to keep doing so.

Reply:
I've made the list explicit (based on https://en.cppreference.com/w/cpp/language/operators). Note that previously operators would be allowed as long as they contained a '=', so 'x=' would have been a valid operator (but doesn't compile obviously). Those will be caught now as well.